### PR TITLE
brew.sh: define minimum required Glibc version

### PR DESCRIPTION
**Pull request by maxim-belkin**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/HEAD/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/brew/blob/HEAD/Library/Homebrew/test/PATH_spec.rb).
- [ ] Have you successfully run `brew style` with your changes locally?
- [ ] Have you successfully run `brew typecheck` with your changes locally?
- [ ] Have you successfully run `brew tests` with your changes locally?
- [ ] Have you successfully run `brew man` locally and committed any changes?

-----

Define minimum required Glibc version in one central location -- `brew.sh`

### CODE REVIEW

File: Library/Homebrew/cmd/vendor-install.sh [hunk 1]
```diff
@@ -37,14 +37,17 @@ check_ldd_version() {
...
   if [[ $(/usr/bin/ldd --version) =~ \ [0-9]\.[0-9]+ ]]
   then
     ldd_version=${BASH_REMATCH[0]// /}
     ldd_version_major=${ldd_version%.*}
     ldd_version_minor=${ldd_version#*.}
-    if (( ldd_version_major < 2 || ldd_version_minor < 13 ))
+    if (( ldd_version_major < minimum_required_major || ldd_version_minor < minimum_required_minor ))
     then
-      odie "Vendored tools require system Glibc 2.13 or later."
+      odie "Vendored tools require system Glibc $HOMEBREW_MINIMUM_GLIBC_VERSION or later (yours is $ldd_version)."
```
Comment (MikeMcQuaid) [inline Library/Homebrew/cmd/vendor-install.sh:50]: Would be good to get this function no-oping either when `HOMEBREW_LINUX` isn't set and perhaps when `HOMEBREW_MINIMUM_GLIBC_VERSION` isn't set to ensure it doesn't fail when called on macOS. Might also be worth renaming to `check_glibc_ldd_version` or `check_linux_glibc_ldd_version` or something

File: Library/Homebrew/brew.sh [hunk 2]
```diff
@@ -410,6 +410,8 @@ Your Git executable: $(unset git && type -p $HOMEBREW_GIT)"
     fi
   fi
 
+  HOMEBREW_MINIMUM_GLIBC_VERSION="2.13"
```
Comment (MikeMcQuaid) [inline Library/Homebrew/brew.sh:413]:
```suggestion
  HOMEBREW_LINUX_MINIMUM_GLIBC_VERSION="2.13"
```

or something to clarify this is only set on Linux?

**Review by MikeMcQuaid — COMMENTED** [review]
Thanks @maxim-belkin! Looking good, just had some ideas on making this robust to future refactoring.

**Review by MikeMcQuaid — APPROVED** [review]
👏🏻 Perfect, thanks @maxim-belkin!

**Comment by maxim-belkin** [comment]
Thanks for the review, Mike!

**Comment by sjackman** [comment]
Thanks, Maxim!

**Pull request merged**